care/facility/api/serializers/patient.py

```diff
@@ -65,11 +66,23 @@ class PatientContactDetailsSerializer(serializers.ModelSerializer):
     mode_of_contact = ChoiceField(choices=PatientContactDetails.ModeOfContactChoices)
 
     patient_in_contact_object = PatientListSerializer(read_only=True, source="patient_in_contact")
+    patient_in_contact = serializers.UUIDField(source="patient_in_contact.external_id")
 
     class Meta:
         model = PatientContactDetails
         exclude = ("patient",)
 
+    def validate_patient_in_contact(self, value):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5498155)